mpas_analysis/shared/timekeeping/utility.py

```diff
@@ -226,4 +482,32 @@ def _parseDateString(dateString, isInterval=False):  # {{{
     return (year, month, day, hour, minute, second)  # }}}
 
 
+def _mpas_to_netcdf_calendar(calendar):
+    """
+    Convert from MPAS calendar to NetCDF4 calendar names.
+    """
+
+    if calendar == 'gregorian_noleap':
+        calendar = 'noleap'
+    elif calendar != 'gregorian':
+        raise ValueError('Unsupported calendar {}'.format(calendar))
+    return calendar
+
+
+def _round_datetime(date):
+    """Round a datetime object to nearest second
+    date : datetime.datetime or similar objet object.
```

Comment:
Do we want / need full doc strings here?  I'm ok with the simple ones presented but thought I'd ask, more so for my own information in the future.